Add `add_observer_metadata` `geo.name` to Quickstart (#21501)
elastic/beats: heartbeat/docs/getting-started.asciidoc

include::{libbeat-dir}/shared/config-check.asciidoc[]

[float]
[[configurelocation]]
=== Step 4: Configure the Heartbeat location

Heartbeat can be deployed in multiple locations so that you can detect 
differences in availability and response times across those locations.

> By configuring Heartbeat with the location of the server you will allow the
> Uptime application to display location specific information on a map, and 
> allow the Uptime anomaly detection to detect anomalies based on location.

Reviewer: ```suggestion
Configure the Heartbeat location to allow {kib} to display location-specific
information on Uptime maps and perform Uptime anomaly detection based
on location.
```

Reviewer: I'm trying to shorten this a bit and use language that's parallel with the heading, but my editing brain's a bit out to lunch. Is this better?

Author: much better! Thanks.
